<?xml version="1.0"?>
<launch>    
    <!-- PARAMETERS FOR CONFIGURING ROBOT BEHAVIOR -->
    <!-- The urgency threshold for the robot policy -->
    <arg name="urgency_threshold" default="20" />
    <param name="urgency_threshold" type="int" value="$(arg urgency_threshold)" />  
    <!-- The list of room which contain a recharge station -->
    <arg name="recharge_rooms" default="['E']" />
    <param name="recharge_rooms" type="str" value="$(arg recharge_rooms)" />
    <!-- The ordered list of room priorities for deciding the next room -->
    <arg name="rooms_priority" default="['URGENT', 'CORRIDOR', 'ROOM']" />
    <param name="rooms_priority" type="str" value="$(arg rooms_priority)" />
    <!-- The time the robot takes to explore a room -->
    <arg name="exploration_seconds" default="2" />
    <param name="exploration_seconds" type="int" value="$(arg exploration_seconds)" />
    <!-- The battery percentage for which the robot needs to recharge it -->
    <arg name="battery_require_recharge" default="20" />
    <param name="battery_require_recharge" type="int" value="$(arg battery_require_recharge)" />
    <!-- The battery percentage for which the robot stops recharging -->
    <arg name="battery_stop_recharge" default="85" />
    <param name="battery_stop_recharge" type="int" value="$(arg battery_stop_recharge)" />    

	<!-- LAUNCHING NODE FOR ROBOT BEHAVIOR-->
    <node name="robot_surveillance" pkg="robot_behavior" type="robot_surveillance.py" output="screen"/>
</launch>
